### ISSUE TITLE
Add overlapping labels in timeline

### ISSUE THREAD

**Issue by oThommy** [Interactive-Telemetry-for-Design/interactive-telemetry-for-design#46]
Overlapping labels in timeline when using sigmoid and correct loss function instead of softmax and categorical_crossentropy.
